exercises/02-robot-syntax.md

```diff
@@ -1,6 +1,7 @@
 # Robot Syntax
 
-In the previous exercise we created `robot/login.robot` file and the content was most likely something like this:
+<details>
+    <summary>In the previous exercise we created <code>robot/login.robot</code> file and the content was most likely something like this</summary>
```

Comment:
Is there a reason for HTML in Markdown?

Reply:
It's the only way to get a collapsable block.

Reply:
Is there a reason to have collapsible box?

Reply:
Just to make the exercise file more tidy overall and by default to show only the steps for the current exercise.